part1_pseudocode.txt

```diff
@@ -0,0 +1,22 @@
+
+Write argparse code to input filename/location as a variable.
+
+Create two empty lists - 
+	-One will contain the alignment positions.
+	-The other will contain UMI/Randomers.
```

Comment:
Creating the two lists works, however think about maybe doing a dictionary instead. Using the UMI's as the keys and the positions as the values. It would make it a little more efficient (though this works just fine)